boolean/boolean.py

```diff
@@ -1161,7 +1161,7 @@ def start_operation(ast, operation):
 
     ast = [None, None]
 
-    for token, tokstr, row, col in tokenized:
+    for token, tokstr, position in tokenized:
```

Comment:
You would need to also update the docstring for `tokenzier` and `parse` to document the new expectations?